tests/platform/cloud/CloudRevisionGetTests.js

```diff
@@ -0,0 +1,120 @@
+/*!
+Tests for CBFM's /revision endpoint
+
+Copyright 2020 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/GPII/universal/blob/master/LICENSE.txt
+*/
+
+"use strict";
+
+var fluid = require("infusion"),
+    gpii = fluid.registerNamespace("gpii"),
+    jqUnit = fluid.registerNamespace("jqUnit");
+
+fluid.require("%gpii-universal");
+
+gpii.loadTestingSupport();
+
+fluid.registerNamespace("gpii.tests.cloudRevision");
+
+gpii.tests.cloudRevision.validGpiiRevision = require(
+    fluid.module.resolvePath(
+        "%gpii-universal/tests/data/gpiiRevision/gpii-revision-valid.json"
```

Comment:
Did you forget to push up this test file?

Reply:
Yes, and CI just informed me via a failure.